evap/staff/tests/test_forms.py

```diff
@@ -1040,16 +1041,14 @@ def test_unused_questionnaire_visibility(self):
     def test_answers_for_removed_questionnaires_deleted(self):
         # pylint: disable=too-many-locals
         evaluation = baker.make(Evaluation)
-        general_question_1 = baker.make(Question, type=QuestionType.POSITIVE_LIKERT)
```

Comment:
I wonder if can make this closer to the original code by using baker's `make_m2m` argument. I haven't checked what exactly it promises though. Still, one would hope that is some way to do something like `baker.make(my_questionnaire.questions, type=...)` or `baker.make(Question, questionnaires__count=1)` or so. Do you know if something like this is possible?

Reply:
I was more thinking about all the `some_question = baker.make(QuestionAssignment, ...)` lines which I find dubious because it's not a question but a question assignment. Actually, today, I think you are right and it's close enough and fine how the tests change here and it's something we can also fixup later if we care.

Outside of tests: How do we feel about having `Answer.question` actually being a question assignment? Should we rename it to `Answer.assignment`? Should we maybe even add `@property def question(self): return self.assignment.question`? Or is there a good reason to keep using the old name?

cc @richardebeling